prymer/offtarget/offtarget_detector.py

```diff
@@ -150,6 +165,19 @@ def __init__(
         executable: str | Path = "bwa",
     ) -> None:
         """
+        Initialize an [[OffTargetDetector]].
```

Comment:
Can the primer pair related parameters be given defaults? I don't need them for checking single primers.

Reply:
I have been wondering something similar - if it's possible to provide defaults for more of these parameters, or if the features should be decoupled

cc @tfenne @emmcauley